Sources/Snowplow/Configurations/TrackerConfiguration.swift

```diff
@@ -264,8 +268,14 @@ public class TrackerConfiguration: SerializableConfiguration, TrackerConfigurati
     /// It is called repeatedly (on each tracked event) until a UUID is returned.
     @objc
     public var advertisingIdentifierRetriever: (() -> UUID?)? {
```

Comment:
is this still needed?

Reply:
It's not important, but it's kind of a convenience function. On the background it uses the new API anyway.